src/js/packages/@reactpy/client/src/vdom.tsx

```diff
@@ -198,23 +203,43 @@ function createEventHandler(
   name: string,
   { target, preventDefault, stopPropagation }: ReactPyVdomEventHandler,
 ): [string, () => void] {
-  return [
-    name,
-    function (...args: any[]) {
-      const data = Array.from(args).map((value) => {
-        if (!(typeof value === "object" && value.nativeEvent)) {
-          return value;
-        }
-        const event = value as React.SyntheticEvent<any>;
-        if (preventDefault) {
-          event.preventDefault();
-        }
-        if (stopPropagation) {
-          event.stopPropagation();
-        }
-        return serializeEvent(event.nativeEvent);
-      });
-      client.sendMessage({ type: "layout-event", data, target });
-    },
-  ];
+  const eventHandler = function (...args: any[]) {
+    const data = Array.from(args).map((value) => {
+      if (!(typeof value === "object" && value.nativeEvent)) {
+        return value;
+      }
+      const event = value as React.SyntheticEvent<any>;
+      if (preventDefault) {
+        event.preventDefault();
+      }
+      if (stopPropagation) {
+        event.stopPropagation();
+      }
```

Comment:
What is your anticipated method for having users modify `preventDefault` and `stopPropagation` under this new system?

Would it not be better to have users handle it exclusively through JavaScript? For example `this.event.preventDefault()`?

Reply:
I didn't add that, that's original code from the standard `eventHandler`. It shows as additions because I did pull it directly out of the return statement as an anonymous function and gave it a name - all so I could tack on a `isHandler = true` property (see line 222 below). This is how I chose to distinguish between the standard `eventHandler` and this new `jsExecutable` in the `react.js` template used for web components:
![image](https://github.com/user-attachments/assets/c24d112c-9784-44f9-844e-812e2e7c4e1e)

Alternatively, I had done `value.toString().includes('.sendMessage')`, but thought that was a bit expensive to do every time. Maybe not?